content/1.4-migration.md

```diff
@@ -64,3 +64,44 @@ if (Meteor.isClient) {
 ```
 
 One place this is particularly useful is in [test files that are only intended to run on the client or the server](https://github.com/meteor/todos/commit/3963a65d96cd7ef235a95d5e3a331d6f0606f70f) &mdash; you can now use `import` wherever you like, without having to organize your tests in `client` or `server` directories.
+
+<h2 id="gotchas">Gotchas!</h2>
+Here is a list of observed gotchas...
+
+<h3 id="mongo-3_2-gotchas">Mongo 3.2</h3>
+
+If you are using [oplog tailing](https://github.com/meteor/docs/blob/version-NEXT/long-form/oplog-observe-driver.md) and you see a failed authentication you may need to upgrade to [SCRAM-SHA-1](https://docs.mongodb.com/manual/release-notes/3.0-scram/#upgrade-mongodb-cr-to-scram), essentially: `use admin, db.adminCommand({authSchemaUpgrade: 1});`.  You may need to delete and re-add your oplog reader user.
+
+<h3 id="node-gyp-gotchas">node-gyp</h3>
+
+Make sure this is installed so you can compile your node modules to native if necessary.  See [Install `node-gyp`](https://github.com/nodejs/node-gyp#installation)
```

Comment:
Hi @tmeasday, I believe I have been confused about meteor interaction with npm for some time.  If you could look at https://forums.meteor.com/t/meteor-and-the-npm-global-repository/27154 and make a comment, I would like to know if I am thinking correctly about it now.

node-gyp is in the meteor global npm repository.

Thanks!